toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon

```diff
@@ -0,0 +1,96 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/no_prelude/bad_import.carbon
+
+// --- fail_import_cpp.carbon
+
+library "[[@TEST_NAME]]";
+
+// CHECK:STDERR: fail_import_cpp.carbon:[[@LINE+4]]:1: error: imported API 'Cpp' not found [ImportNotFound]
+// CHECK:STDERR: import Cpp;
+// CHECK:STDERR: ^~~~~~
+// CHECK:STDERR:
+import Cpp;
+
+// --- fail_import_cpp_library.carbon
+
+library "[[@TEST_NAME]]";
+
+// CHECK:STDERR: fail_import_cpp_library.carbon:[[@LINE+8]]:19: error: expected a string literal to specify the library name [ExpectedLibraryName]
+// CHECK:STDERR: import Cpp library;
+// CHECK:STDERR:                   ^
+// CHECK:STDERR:
+// CHECK:STDERR: fail_import_cpp_library.carbon:[[@LINE+4]]:19: error: semantics TODO: `handle invalid parse trees in `check`` [SemanticsTodo]
+// CHECK:STDERR: import Cpp library;
+// CHECK:STDERR:                   ^
+// CHECK:STDERR:
+import Cpp library;
+
+// --- fail_import_cpp_library_empty.carbon
+
+library "[[@TEST_NAME]]";
+
+// CHECK:STDERR: fail_import_cpp_library_empty.carbon:[[@LINE+4]]:1: error: imported API 'Cpp' not found [ImportNotFound]
```

Comment:
This is a bit weird, the same diagnostic as `import Cpp;`. I was expecting `file '' not found`. Maybe we can improve this in a followup.

Reply:
We now diagnose with `Cpp` import missing library